diff --git a/node_modules/@atproto/api/dist/agent.js b/node_modules/@atproto/api/dist/agent.js
index 8d2744e..8f0385e 100644
--- a/node_modules/@atproto/api/dist/agent.js
+++ b/node_modules/@atproto/api/dist/agent.js
@@ -590,7 +590,7 @@ class Agent extends xrpc_1.XrpcClient {
                 hideBadges: false,
             },
         };
-        const res = await this.app.bsky.actor.getPreferences({});
+        const res = await this.app.bsky.actor.getPreferences({}, {headers: {'atproto-proxy': ''}});
         const labelPrefs = [];
         for (const pref of res.data.preferences) {
             if (predicate.isValidAdultContentPref(pref)) {
@@ -1237,14 +1237,14 @@ class Agent extends xrpc_1.XrpcClient {
     async updatePreferences(cb) {
         try {
             await __classPrivateFieldGet(this, _Agent_prefsLock, "f").acquireAsync();
-            const res = await this.app.bsky.actor.getPreferences({});
+            const res = await this.app.bsky.actor.getPreferences({}, {headers: {'atproto-proxy': ''}});
             const newPrefs = cb(res.data.preferences);
             if (newPrefs === false) {
                 return res.data.preferences;
             }
             await this.app.bsky.actor.putPreferences({
                 preferences: newPrefs,
-            });
+            }, {headers: {'atproto-proxy': ''}});
             return newPrefs;
         }
         finally {
diff --git a/node_modules/@atproto/api/dist/moderation/decision.js b/node_modules/@atproto/api/dist/moderation/decision.js
index ef44b3d..f6c9f18 100644
--- a/node_modules/@atproto/api/dist/moderation/decision.js
+++ b/node_modules/@atproto/api/dist/moderation/decision.js
@@ -67,6 +67,8 @@ class ModerationDecision {
     ui(context) {
         const ui = new ui_1.ModerationUI();
         for (const cause of this.causes) {
+            if (cause?.label?.val === '!no-unauthenticated') continue;
+
             if (cause.type === 'blocking' ||
                 cause.type === 'blocked-by' ||
                 cause.type === 'block-other') {
